### ISSUE TITLE
community simulation media

### ISSUE THREAD

**Issue by H1889** [jotech/gapseq#125]
Hello, I am going to simulate a community with 3 bacteria, A,B and C. B takes the products of A and C takes the products of B. When generating the models for gapfilling which medium do I use? Do I use the medium of A for all species or  the recommended medium for each species?
Thanks

**Comment by H1889** [comment]
I will do it. Thanks

**Comment by H1889** [comment]
Sorry again, but I have another question, when running R with de gapfilled models with each predicted medium it is necesarry to supply a medium (arena_subs <- fread("medium.csv"), which medium is the best? I suppose the medium for A speices, is it not?
Thanks

**Comment by jotech** [comment]
I'm not sure if I get what you want to do. But if you want to use the predicted media but do not want to merge the media for the community simulation, I would propose to use a standard medium defined by essential metabolites from all predicted media plus individual carbon/energy sources and minus the compounds that some members of the community can produce. Does it somehow answer your question?

**Comment by H1889** [comment]
Yes you are right, but I wonder if we should not also simulate real life. In real life the input medium must contain the inorganic compunds, amino acids and vitamins requiered for all species plus the carbon source of species A and not the other carbon sources of the other species. Is that right?
Thanks for the help.
